virtio-devices/src/block.rs

```diff
@@ -232,13 +258,43 @@ impl BlockEpollHandler {
                 .ok_or(Error::MissingEntryRequestList)?;
             request.complete_async().map_err(Error::RequestCompleting)?;
 
+            let latency = request.start.elapsed().as_micros() as u64;
+            let read_ops_lst = self.counters.read_ops.load(Ordering::Relaxed) as i64;
```

Comment:
What's `lst` ?

Reply:
Oh, I think you shortened "last"? Generally I would call it `old_read_ops`

Reply:
Yes, its 'last', wiil update